app.js

```diff
@@ -0,0 +1,22 @@
+import mu from 'mu';
+
+const app = mu.app;
+const bodyParser = require('body-parser');
+const repository = require('./repository');
+const cors = require('cors');
+const debug = process.env.DEBUG_LOGGING || false;
+
+app.use(bodyParser.json({ type: 'application/*+json' }));
```

Comment:
Body parsing van payloads met MIME-type voor JSONAPI wordt al [afgehandeld door de template](https://github.com/mu-semtech/mu-javascript-template/blob/335321a489fa432e859dfd2a35146eb3e14e7021/helpers/mu/server.js#L8). In het geval je gewone json payloads zou willen parsen gaat dit ook niet werken, want da's `application/json` (zonder de `+`). Deze lijn heb je dus niet echt nodig.

Reply:
Solved, alle dependencies zijn gecleared.